docs/src/lib/examples/kitchen-sink/demos/Default.svelte

```
@@ -0,0 +1,763 @@
+    const { groupByIds } = pluginStates.group
+    const { sortKeys } = pluginStates.sort
+    const { filterValues } = pluginStates.filter
+    const { filterValue } = pluginStates.tableFilter
+    const { selectedDataIds } = pluginStates.select
+    const { pageIndex, pageCount, pageSize, hasPreviousPage, hasNextPage } = pluginStates.page
+    const { expandedIds } = pluginStates.expand
+    const { columnIdOrder } = pluginStates.orderColumns
+    const { hiddenColumnIds } = pluginStates.hideColumns
+    const { columnWidths } = pluginStates.resize
+
+    let hideForId: Record<string, boolean> = $state(
```

ESLint

[error] 'hideForId' is never reassigned. Use 'const' instead.

(prefer-const)